R/make_subset_expr.R

```diff
@@ -259,18 +267,6 @@ create_colinfo <- function(lyt, df, rtpos = TreePos(),
       0L
     } else {
       vec <- try(eval(ex, envir = alt_counts_df), silent = TRUE)
-      if (is(vec, "try-error")) {
```

Comment:
why removed?

Reply:
damnit. apparently whe you type in here then hit resolve it doesn't make the comment.... ugh.

Anyway, this check is now done during coltree construction. Previously it was here because coltree construction only had access to df, but now it has access to and uses both df and alt_counts_df, so the check occurs more naturally there.